axon-server-connector/src/test/java/org/axonframework/axonserver/connector/heartbeat/HeartbeatConnectionCheckTest.java

```diff
@@ -0,0 +1,68 @@
+package org.axonframework.axonserver.connector.heartbeat;
+
+import org.junit.*;
+
+import java.time.Instant;
+import java.util.concurrent.atomic.AtomicReference;
+
+import static java.time.temporal.ChronoUnit.MILLIS;
+import static java.time.temporal.ChronoUnit.SECONDS;
+import static org.junit.Assert.*;
+
+/**
```

Comment:
How ever small, I think this block would benefit from a one-line description what this class is for.